src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -18,6 +18,10 @@ <H2>ASP.NET ViewState Integrity</H2>
 <H2>Base64 Disclosure</H2>
 Base64 encoded data was disclosed by the application/web server
 
+<H2>Link uses _blank target</H2>
+This checks to see if any links use a target of "_blank" without  without using either keyword "noopener" or "noreferrer", as this will allow target pages to take over the page that opens them.<br>
+By default this rule will ignore all links that are in the same context as the page and at the LOW threshold will raise alerts for that link to other hosts in the context.
```

Comment:
This isn’t quite right. Context is only checked at low, perhaps the first part of the sentence should say “same hostname”?

Reply:
Context is being checked if not low?

Reply:
Doh missed the bang